lib/reporters/tap.js

```diff
@@ -52,11 +53,18 @@ function TAP(runner) {
   runner.on('fail', function(test, err) {
     failures++;
     console.log('not ok %d %s', n, title(test));
-    if (err.message) {
-      console.log(err.message.replace(/^/gm, '  '));
-    }
-    if (err.stack) {
-      console.log(err.stack.replace(/^/gm, '  '));
+    var emitYamlBlock = err.message != null || err.stack != null;
```

Comment:
I see "diff"-able `Error`s in Mocha codebase with `actual` and `expected` properties. Would they be considered "interesting" as output?

Reply:
I think this is a great feature to have but it doesn't really have much to do with TAP13, rather a "while we're at it" thing and if I remember correctly I ran into some problems investigating it, but this was I while back. I think this feature shouldn't ride on this PR tbh.

Reply:
But wasn't that _exactly_ part of the `tape` output?